### PR TITLE
Release notes script

### PR DESCRIPTION
**Pull request by Jrice1317**
### Description

This PR will create release notes for Miniconda from `constructor` outputs. 

Changes:
- Add Release Notes script and template
- Add data for `Miniconda 23.5.0-3` used to generate Release Notes for current release
- Add link to Release Notes to `miniconda.rst`

The screenshot below is a snippet of what the rst file generated by the script looks like.
Screen Shot 2023-07-11 at 2 45 10 PM

**Contributors:**
- [Jaida Rice]( @Jrice1317)
- [Marco Esters](@marcoesters) 
- [Steve Crass](@SC426)

### Checklist - did you ...

- [ - ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ - ] Add / update necessary tests?
- [ - ] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/miniconda_release_notes.rst [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+
+Miniconda 23.5.0-3 (July 11, 2023)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+
+User-Facing Changes
+-------------------
+
+* Conda has been updated to v23.5.0
+* OpenSSL has been updated to version 3
+* The default installer uses python-3.11.3
```
Comment (kathatherine) [inline docs/source/miniconda_release_notes.rst:13]:
```suggestion
* Conda has been updated to v23.5.0.
* OpenSSL has been updated to version 3.
* The default installer uses python-3.11.3.
```

File: docs/source/miniconda_release_notes.rst [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+
+
+
+Available Python Versions:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+Packages:
+^^^^^^^^^
```
Comment (kathatherine) [inline docs/source/miniconda_release_notes.rst:544]: This looks like it got doubled up.

File: docs/source/miniconda_releases/23.5.0-3/changes.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
+User-Facing Changes
+-------------------
+
+* Conda has been updated to v23.5.0
+* OpenSSL has been updated to version 3
+* The default installer uses python-3.11.3
```
Comment (kathatherine) [inline docs/source/miniconda_releases/23.5.0-3/changes.rst:6]:
```suggestion
* Conda has been updated to v23.5.0.
* OpenSSL has been updated to version 3.
* The default installer uses python-3.11.3.
```

**Review by kathatherine — CHANGES_REQUESTED** [review]
Just a couple things!

**Comment by kenodegard** [comment]
@conda-bot check

**Review by kathatherine — DISMISSED** [review]

**Review by kathatherine — DISMISSED** [review]

File: docs/source/miniconda_release_notes.rst.jinja2 [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+{{ heading }}
+{{ "^" * heading|length }} 
+
+{% if not release.notes -%}
+{%- else -%}
+{{ release.notes }}
+{%- endif %}
+
+Available Python Versions:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (marcoesters) [inline docs/source/miniconda_release_notes.rst.jinja2:17]:
```suggestion
Available Python Versions
--------------------------
```

File: docs/source/miniconda_release_notes.rst.jinja2 [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+{% for py_version in release.python_versions -%}
+* {{ py_version }}
+{% endfor %}
+
+Packages:
+^^^^^^^^^
```
Comment (marcoesters) [inline docs/source/miniconda_release_notes.rst.jinja2:24]:
```suggestion
Packages
----------
```

File: docs/source/miniconda_release_notes.rst [hunk 6]
```diff
@@ -0,0 +1,533 @@
...
+
+
+Known Issues
+------------
+
+* The .pkg installers may skip the "Destination Select" page after accepting the license agreement. See our `Anaconda installation instructions <https://docs.anaconda.com/free/anaconda/install/mac-os/>`_ for details.
+
+
+Available Python Versions:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (marcoesters) [inline docs/source/miniconda_release_notes.rst:29]:
```suggestion
Available Python Versions
--------------------------
```

File: docs/source/miniconda_release_notes.rst [hunk 7]
```diff
@@ -0,0 +1,533 @@
...
+
+* 3.11
+* 3.10
+* 3.9
+* 3.8
+
+
+Packages:
+^^^^^^^^^
```
Comment (marcoesters) [inline docs/source/miniconda_release_notes.rst:38]:
```suggestion
Packages
----------
```

**Review by marcoesters — COMMENTED** [review]
I think the headers should have consistency, or the font sizes will keep changing inside each release note

**Review by marcoesters — APPROVED** [review]
Thank you!

**Pull request merged**